### PR TITLE
add admin configuration of file formats

### PR DESCRIPTION
**Pull request by leonidastri**
As a functional admin I want to configure lists of preservable file formats so that I ensure publication terms are up to date.

Features added:

As a functional admin the admin can navigate to Administration page and:

they can now upload a new file of file formats.
they can check the current list of files of file formats.
they can choose a file of file formats and delete it. Then, the file is moved to trash folder, and there is a possibility to retrieve it in case of wrong deletion.

### CODE REVIEW

**Comment by leonidastri** [comment]
This should be reviewed in combination with the following pr in yoda-portal:
https://github.com/UtrechtUniversity/yoda-portal/pull/368

File: roles/yoda_rulesets/tasks/yoda-ruleset.yml [hunk 1]
```diff
@@ -320,6 +320,15 @@
   when: (not ansible_check_mode) and (formats_installed.stderr.find('does not exist') >= 0)
 
 
+- name: Ensure admin has permissions to make changes to file formats
+  become_user: "{{ irods_service_account }}"
+  become: true
+  # noqa no-changed-when
+  ansible.builtin.command:
+    ichmod -rM own priv-admin /{{ irods_zone }}/yoda/file_formats
+  when: priv_groups.stdout.find('priv-admin') != -1
```
Comment (lwesterhof) [inline roles/yoda_rulesets/tasks/yoda-ruleset.yml]: We can add `not ansible_check_mode` to prevent it from running in check mode.

**Review by lwesterhof — CHANGES_REQUESTED** [review]
Would be nice to add an extra task to checs if priv-admin already has permissions, otherwise this task will run every time the playbook runs.

File: roles/yoda_rulesets/tasks/yoda-ruleset.yml (hunk 1 above)
Comment (leonidastri) [reply roles/yoda_rulesets/tasks/yoda-ruleset.yml]: added check

**Comment by leonidastri** [comment]
Updated the code considering the comments!

**Review by lwesterhof — APPROVED** [review]
LGTM!

**Pull request merged**